src/utils/index.ts

```diff
@@ -295,6 +294,41 @@ export const fetchTranscriptDetails = (allTranscripts: Transcript[], paths: stri
   };
 };
 
+export const createLanguageTreeArray = (languageTree: any) => {
+  const getValues = Object.entries(languageTree).map(([key, value]) => {
+    const valDetails = (value as unknown as any).data;
+
+    const extractCount = (arr: {} | []) => {
+      let count: number = 0;
+
+      if (Array.isArray(arr)) {
+        count = arr.length;
+      } else {
+        let initialCount = 0;
+
+        const keysPresent = Object.keys(valDetails);
+        keysPresent.forEach((key) => {
+          const item = valDetails[key]?.data.length;
+          initialCount += item;
+        });
+        count = initialCount;
+      }
+
+      return count;
+    };
+
+    const getSourceDetails = {
```

Comment:
we can simply return the object rather than instantiate then return.
```tsx
return {
      slug: key,
      name: (value as unknown as any)?.metadata.title,
      count: extractCount(valDetails)
}
```